src/sourmash/sourmash_args.py

```diff
@@ -391,14 +391,22 @@ def load_pathlist_from_file(filename):
     try:
         with open(filename, 'rt') as fp:
             file_list = [ x.rstrip('\r\n') for x in fp ]
-
-        if not os.path.exists(file_list[0]):
-            raise ValueError("first element of list-of-files does not exist")
+        if len(file_list) == 0:
+            raise ValueError("list-of-files is empty")
+        cnt = 0
+        for i in range(len(file_list)):
+            if not os.path.exists(file_list[i]):
+                cnt = cnt + 1
+        if cnt > 0:
+            raise ValueError("list-of-files contains a badly formatted file")
```

Comment:
Is this error message right?  You're checking if the file exists; if I were a user I would not know that "badly formatted" means "does not exist"

Also, suggest using "pathlist" rather than "list of files".

Reply:
Could you suggest a way to check for badly formatted files?

Reply:
I think it's probably fine to just say, "file '<bad stuff in here>' does not exist".